mender-deb-package

```diff
@@ -246,32 +248,38 @@ EOF
         # we set -W no-psabi
         # https://gcc.gnu.org/gcc-7/changes.html
         if [ "${OS_CODENAME}" = "buster" ]; then
-          PATH="/opt/cross-pi-gcc-8.3.0-0/bin:$PATH" \
-              CROSS_COMPILE="arm-linux-gnueabihf" \
-              CC="$CROSS_COMPILE-gcc" \
-              CXX=${CROSS_COMPILE}-g++ \
-              CXXFLAGS=" -Wno-psabi" \
-              GOARCH=arm \
-              GOARM=6 \
-              dpkg-buildpackage -aarmhf \
-              ${sign_flags} \
-              "${dpkg_build_ignore_flag}" \
-              ${skip_patches_flag} \
+          SYSROOT="/opt/cross-pi-gcc-8.3.0-0/arm-linux-gnueabihf/libc/"
+          PATH="/opt/cross-pi-gcc-8.3.0-0/bin:$PATH"         \
+              PKG_CONFIG_PATH="${SYSROOT}usr/lib/pkgconfig/" \
+              PKG_CONFIG_SYSROOT_DIR="${SYSROOT}"            \
+              CROSS_COMPILE="arm-linux-gnueabihf"            \
+              CC="$CROSS_COMPILE-gcc"                        \
+              CXX=${CROSS_COMPILE}-g++                       \
+              CXXFLAGS=" -Wno-psabi"                         \
+              GOARCH=arm                                     \
+              GOARM=6                                        \
+              dpkg-buildpackage -aarmhf                      \
+              ${sign_flags}                                  \
+              "${dpkg_build_ignore_flag}"                    \
+              ${skip_patches_flag}                           \
               --build=$DEB_BUILD_TYPE
         fi
         if [ "${OS_CODENAME}" = "bullseye" ]; then
-          # For some reason CMake find_package does not like it when this dir is missing
-          mkdir /opt/cross-pi-gcc-10.3.0-0/arm-linux-gnueabihf/libc/include
-          PATH="/opt/cross-pi-gcc-10.3.0-0/bin:$PATH" \
-              CROSS_COMPILE="arm-linux-gnueabihf" \
-              CC="$CROSS_COMPILE-gcc" \
-              CXX=${CROSS_COMPILE}-g++ \
-              GOARCH=arm \
-              GOARM=6 \
-              dpkg-buildpackage -aarmhf \
-              ${sign_flags} \
-              "${dpkg_build_ignore_flag}" \
-              ${skip_patches_flag} \
+            # For some reason CMake find_package does not like it when this dir is missing
+            SYSROOT="/opt/cross-pi-gcc-10.3.0-0/arm-linux-gnueabihf/libc/"
+            mkdir /opt/cross-pi-gcc-10.3.0-0/arm-linux-gnueabihf/libc/include
```

Comment:
Why `bullseye` needs an explicit `mkdir` here while `buster` does not? Could we just do it in both places (with `mkdir -p`) for consistency?

Reply:
Suggested beautification:
```suggestion
            mkdir "${SYSROOT}/include"
```